voice_assistant_pkg/package.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>voice_assistant_pkg</name>
```

Comment:
conventionally, packages don't have `pkg` in the name (so just `voice_assistant`)